pkgs/development/python-modules/morecantile/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
```

Comment:
I suggest to alphabetically reorder function parameters and visually split them to functions and packages as we have in [other packages](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/rasterio/default.nix).